src/components/AppHeader.vue

```diff
@@ -1,95 +1,58 @@
 <script lang="ts" setup>
-import { computed, onMounted, ref, onUnmounted } from 'vue'
-import { useRoute, useRouter } from 'vue-router'
+import { onMounted, ref, onUnmounted } from 'vue'
+import { useRouter } from 'vue-router'
 
-import AccountUserIcon from '@/assets/iconography/account-user.svg'
 import ArcanaLogo from '@/assets/iconography/arcana-dark-vertical.svg'
-import ArcanaFavicon from '@/assets/iconography/arcana-favicon.svg'
+import BlueCloseIcon from '@/assets/iconography/close.svg'
 import MenuIcon from '@/assets/iconography/menu.svg'
 import CloseIcon from '@/components/icons/CloseIcon.vue'
-import VHeader from '@/components/lib/VHeader/VHeader.vue'
-import { useAuthStore } from '@/stores/auth.store'
-import { useAppId } from '@/use/getAppId'
-import constants from '@/utils/constants'
 
-const route = useRoute()
 const router = useRouter()
-const authStore = useAuthStore()
-const appId = useAppId()
 const canShowBanner = ref(true)
 const hideHeader = ref(false)
+const showHelpMenu = ref(false)
+const showMenu = ref(false)
 
 let lastScrollTop = 0
 const scrollDelta = 10
 
-const menuItems = computed(() => {
-  let arr = [
-    {
-      label: 'Apps',
-      action() {
-        router.push({ name: 'ManageApps' })
-      },
-      selected: false,
+const helpItems = [
+  {
+    label: 'View Docs',
+    action() {
+      router.push({ name: 'ManageApps' })
     },
-    {
-      label: 'Dashboard',
-      action() {
-        router.push({ name: 'Dashboard', params: { appId } })
-      },
-      selected: false,
-      disabled: false,
+    selected: false,
+  },
+  {
+    label: 'Schedule a Demo',
+    action() {
+      router.push({ name: 'ManageApps' })
```

Comment:
We can directly open the calendly link here: https://calendly.com/arcana-network/arcana-demo-walkthrough?month=2023-01